docs/spec_draft.md

```diff
@@ -2311,6 +2313,49 @@ More formally, for each element `x`: `imag(x) = is_complex(x) ? x.imag : 0.0`.
 
 [Back to Ops](#index-of-ops)
 
+
```

Comment:
Similarly to https://github.com/openxla/stablehlo/pull/580/files#r1037781796, let's introduce another section under "Parallel execution":

```
### Streaming communication

Every StableHLO process has access to two streaming interfaces:
  * **Infeed** that can be read from.
  * **Outfeed** that can be written to.

Unlike channels, which are used to communicate between processes and therefore
have processes at both of their ends, infeeds and outfeeds have their other
end implementation-defined.

Further formalization, e.g. how streaming communication influences execution
order and what kind of synchronization is introduced by it, is TBD.
```